setup.py

```diff
@@ -77,15 +95,76 @@ def finalize_options(self):
         ]
         super().finalize_options()
 
+    def add_optional_flags(self):
```

Comment:
I guess this could be called `get_build_envvars()` or something like that?

Reply:
Renamed to `add_optimization_flags`.